near-fetch/src/lib.rs

```diff
@@ -116,11 +119,19 @@ impl Client {
                     actions: actions.clone(),
                 }
                 .sign(signer.as_signer()),
+                wait_until: wait_until_param,
             })
             .await;
 
         self.check_and_invalidate_cache(&cache_key, &result).await;
-        result.map_err(Into::into)
+
+        let outcome = result
+            .map_err::<Error, _>(Into::into)
```

Comment:
Is this right? `map_err`? Ran into a ton of issues getting this to compile correctly.